File: Leetcode/TwoSum.js
var twoSum = function(nums, target) {
     for (let i = 0; i < nums.length; i++) {
         const compliment = target - nums[i];
         if (compliment in map) {
             return [map[compliment], i]
         }
         map[nums[i]] = i
     }
 };

File: Leetcode/TwoSum.py
class solution:
    def twosum(self, nums:list[int], target:int) -> list[int]:
        hashmap = {}
        for i in range(len(nums)):
            x = target - nums[i]
            if x in hashmap:
                return [map[x], i]
            else:
                hashmap[nums[i]] = i
        return []
        
